ci/docker_build_ci.yml

```diff
@@ -0,0 +1,77 @@
+name: docker_build_ci
+on:
+  push:
+  # everyday at 3 am
+  schedule:
+    - cron:  '0 3 * * *'
```

Comment:
We don't put `schedule` in our starter workflow, preferring either `push` or `release`.